backend/pricing/pricing.go

```diff
@@ -30,12 +30,12 @@ const (
 )
 
 func GetWorkspaceMembersMeter(DB *gorm.DB, workspaceID int) int64 {
-	return DB.Model(&model.Workspace{Model: model.Model{ID: workspaceID}}).Association("Admins").Count()
+	return DB.WithContext(context.TODO()).Model(&model.Workspace{Model: model.Model{ID: workspaceID}}).Association("Admins").Count()
```

Comment:
Just curious: whats the impact on the trace when we use `context.TODO()`?

Reply:
I think it's equivalent to the current behavior where it creates a new trace + context when we don't pass one, but I'm not 100% certain. Curious if @Vadman97 has thoughts on this!